src/types/importCandidate.types.ts

```diff
@@ -29,8 +29,12 @@ export interface ImportCandidate extends Registration {
   importStatus: ImportStatus;
 }
 
-export interface ImportCandidateAggregations {
-  importStatus: AggregationValue<ImportCandidateStatus>[];
+export type CollaborationType = 'Collaborative' | 'NonCollaborative';
+
+export interface ImportCandidateAggregations
+  extends Pick<RegistrationAggregations, 'type' | 'topLevelOrganization' | 'files'> {
```

Comment:
`files` heter egentlig `filesStatus` for importkandidater nå. Men det vil ganske snart endres, så jeg bruker kommende verdi her, sånn at den automatisk dukker opp uten at vi må gjøre noen mer ekstra arbeid på frontend her senere